src/libraries/System.Net.Sockets/src/System/Net/Sockets/UnixDomainSocketEndPoint.cs

```diff
@@ -120,6 +130,24 @@ public override string ToString()
             }
         }
 
+        internal UnixDomainSocketEndPoint CreateBoundEndPoint()
+        {
+            if (IsAbstract(_path))
+            {
+                return this;
+            }
+            return new UnixDomainSocketEndPoint(_path, Path.GetFullPath(_path));
```

Comment:
@antonfirsov the `UnixDomainSocketEndPoint_RelativePathDeletesFile` test fails on Windows like this:
```
Child exception:
  System.Net.Sockets.SocketException (10047): An address incompatible with the requested protocol was used.
   at System.Net.Sockets.Socket..ctor(AddressFamily addressFamily, SocketType socketType, ProtocolType protocolType) in /_/src/libraries/System.Net.Sockets/src/System/Net/Sockets/Socket.cs:line 90
   at System.Net.Sockets.Tests.UnixDomainSocketTest.<>c.<UnixDomainSocketEndPoint_RelativePathDeletesFile>b__15_0() in /_/src/libraries/System.Net.Sockets/tests/FunctionalTests/UnixDomainSocketTest.cs:line 474

Child process:
  System.Net.Sockets.Tests, Version=6.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51 System.Net.Sockets.Tests.UnixDomainSocketTest+<>c Void <UnixDomainSocketEndPoint_RelativePathDeletesFile>b__15_0()
```

I think it doesn't like the `Path.GetFullPath` that we're doing here. Can you figure out what we should do instead?

Reply:
I need to pull down the code to check what's happening, hopefully I'll be able to it within one or two days.

Reply:
@tmds it fails only on older Windows. You should uncomment `    [ConditionalFact(nameof(PlatformSupportsUnixDomainSockets))]`

Reply:
Thanks for figuring this out!